ui/CMakeLists.txt

```diff
@@ -35,3 +35,30 @@ install(CODE
 
 install(PROGRAMS razer-gamewrapper
 	DESTINATION bin)
+
+configure_file("razercfg.desktop.template" "razercfg.desktop")
+
+install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/razercfg.desktop
```

Comment:
SOURCE_DIR is wrong here, because razercfg.desktop is a generated file that will be put into the BINARY_DIR. This breaks, if razercfg is built out of source tree.

Reply:
Fixed.

Reply:
It still does not work. Please do a build-out-of-sourcetree test.
```
CMake Error at ui/cmake_install.cmake:68 (file):
  file INSTALL cannot find
  "/.../razer/build/razercfg.desktop".
```

Reply:
Done.

Reply:
I made that change based off https://github.com/mbuesch/razer/blob/master/ui/CMakeLists.txt#L10

Reply:
Thanks.
I added a few more changes. It would be good if you could test these with your build scripts.

Reply:
Works here, but it appears the applet icon isn't working since the switch to PyQt5.

Reply:
The applet works for me.
What failure do you see?

Reply:
The icon is not displayed.

Python 3.7 and PyQt5 5.11.2

Reply:
FWIW the icon works here on Python 3.7 when built from the commit prior to PyQt5 support.

Reply:
I just tested: With Python 3.7 it also works here.
Are the icons installed correctly on your system?

Reply:
The icon files always end up in the same location, unless I missed something.

Reply:
The icon load mechanism has not been touched in the PySide->Qt5 switch.
So I don't know what's going on.

Reply:
For some reason it's looking for the icons in `/usr/local`..

Reply:
There are some hardcoded search paths:
/usr/share/icons/hicolor/scalable/apps/razercfg.svg
/usr/local/share/icons/hicolor/scalable/apps/razercfg.svg

https://github.com/mbuesch/razer/blob/master/ui/pyrazer/ui.py#L712

Yes, the hardcoded paths are less than ideal and we should use setuptools or something to locate the icons.
But I still don't see how this breaks for you.

Reply:
Might be this: https://bugreports.qt.io/browse/QTBUG-31762